webapp/src/pages/ResponsePage/ResponsePage.js

```diff
@@ -114,42 +143,57 @@ class ResponsePage extends Component {
         };
     };
 
+
     renderReviewResponse(review_response, section) {
         const questions = section.review_questions.map(q => {
             const a = review_response.scores.find(a => a.review_question_id === q.id);
-
+    
             if (a) {
-                return <div>
-            <div key={q.question_id} className="question-answer-block">
-                <p><span className="question-headline">{q.headline}</span>
-                    {q.description && a && <span className="question-description"><br/>{q.description}</span>}
-                </p>
-                <h6><AnswerValue answer={a} question={q} /></h6>
-            </div>
-            </div>
+                return (
+                    <div key={q.id} className="question-answer-block">
+                        <p>
+                            <span className="question-headline">{q.headline}</span>
+                            {q.description && <span className="question-description"><br />{q.description}</span>}
+                        </p>
+                        <h6><AnswerValue answer={a} question={q} /></h6>
+                    </div>
+                );
             }
-            
-            
+            return null;
         });
-        return questions
-    };
     
-    // Render Reviews
-    renderCompleteReviews(){
+        return questions;
+    }
+    
+    renderCompleteReviews() {
         if (this.state.reviewResponses.length) {
-                const reviews = this.state.reviewResponses.map(val => {
-                    return <div className="section">
-                            <h4
-                                className="reviewer-section" >
-                                {this.props.t(val.reviewer_user_firstname + " " + val.reviewer_user_lastname)}
-                            </h4>
-                        {this.renderReviewResponse(val, this.state.reviewForm.review_sections[0])}
+            const reviews = this.state.reviewResponses.map(val => {
+                return (
+                    <div key={val.id} className="review-container">
+                        <h4 className="reviewer-section">
+                            {this.props.t(val.reviewer_user_firstname + " " + val.reviewer_user_lastname)}
+                        </h4>
+                        {this.state.reviewForm.review_sections.map(section => (
+                            <div key={section.id} className="section">
+                            <div className="flex baseline"><h3>{section.headline}</h3></div>
+                                {this.renderReviewResponse(val, section)}
+                            </div>
+                        ))}
+                        <hr/>
+                        <div className="review-total-score">
+                        <h5>Total Score: {this.state.totalScore}</h5>
```

Comment:
I think this should be the total score of the review that is being rendered, no?

Reply:
Indeed